<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FntSetting</class>
 <widget class="QWidget" name="FntSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="2,0,3,0,3,0,2,0,3">
   <property name="topMargin">
    <number>9</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="OverallContainer">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="OverallTitle">
       <property name="font">
        <font>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Overall MCQ Settings : </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="BoostStemSizeContainer">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="BoostStemLabel">
         <property name="text">
          <string>Increase stem's size by : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="BoostStemValueLabel">
         <property name="text">
          <string>20</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="BoostStemSlider">
         <property name="maximum">
          <number>20</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="HiraganaContainer">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="HiraganaTitle">
       <property name="font">
        <font>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Hiragana Settings : </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="HiraganaFtSubContainer">
       <item>
        <widget class="QLabel" name="HiraganaFntLabel">
         <property name="text">
          <string>Font : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="HiraganaFntDropdown">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="HiraganaSizeSubContainer">
       <item>
        <widget class="QLabel" name="HiraganaSizeLabel">
         <property name="text">
          <string>Size : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="HiraganaSizeValueLabel">
         <property name="text">
          <string>15</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="HiraganaSizeSlider">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>20</number>
         </property>
         <property name="maximum">
          <number>150</number>
         </property>
         <property name="value">
          <number>20</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="KatakanaContainer">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="KatakanaTitle">
       <property name="font">
        <font>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Katakana Settings :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="KatakanaFntSubContainer">
       <item>
        <widget class="QLabel" name="katakanaFntLabel">
         <property name="text">
          <string>Font  : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="KatakanaFntDropdown">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="KatakanaSizeSubContainer">
       <item>
        <widget class="QLabel" name="KatakanaSizeLabel">
         <property name="text">
          <string>Size : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="KatakanaSizeValueLabel">
         <property name="text">
          <string>15</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="KatakanaSizeSlider">
         <property name="minimum">
          <number>20</number>
         </property>
         <property name="maximum">
          <number>150</number>
         </property>
         <property name="value">
          <number>20</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="KanjiContainer">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Kanji Settings :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="KanjiSubContainer">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="KanjiFntLabel">
         <property name="text">
          <string>Font :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="KanjiFntDropdown"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="RomanjiContainer">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="RomanjiTitle">
       <property name="font">
        <font>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Romanji Settings : </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="RomanjiFntSubContainer">
       <item>
        <widget class="QLabel" name="RomanjiFntLabel">
         <property name="text">
          <string>Font : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="RomanjiFntDropdown">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="RomanjiSizeSubContainer">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="RomanjiSizeLabel">
         <property name="text">
          <string>Size : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="RomanjiSizeValueLabel">
         <property name="text">
          <string>15</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="RomanjiSizeSlider">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>20</number>
         </property>
         <property name="maximum">
          <number>150</number>
         </property>
         <property name="value">
          <number>20</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
